README.md

```diff
@@ -81,10 +81,10 @@ pushNotifications.publishToUsers(["jonathan", "jordan", "luís", "luka", "mina"]
 })
 
 // Authenticate User
-pushNotifications.authenticateUser("Elon M", completion: { result in
+pushNotifications.generateToken("Elon M", completion: { result in
     switch result {
-    case .value(let jwtTokenString):
-        print("\(jwtTokenString)")
+    case .value(let jwtToken):
+        print("\(jwtToken)")
```

Comment:
How will this print now that its a dictionary?

Reply:
e.g. Do we have any language built-in way to suggest them to marshal to JSON?

Reply:
This is just an example.